lib/container/container.go

```diff
@@ -133,6 +133,25 @@ func WaitContainer(id string) (int, error) {
 	return client.WaitContainer(id)
 }
 
+func getNewDockerClient() (*docker.Client, error) {
+	if os.Getenv("DOCKER_TLS_VERIFY") == "1" {
+		path := os.Getenv("DOCKER_CERT_PATH")
```

Comment:
I didn't do checks for files on purpose, as `go-dockerclient` will make some and return the proper error.

Setting the path as `~/.docker` is interesting, I'll start by this and if you really want me to do the file checks, I'll do them.